landoapi/email.py

```diff
@@ -4,12 +4,14 @@
 import logging
 from email.message import EmailMessage
 
+from landoapi.validation import REVISION_ID_RE
+
 logger = logging.getLogger(__name__)
 
 LANDING_FAILURE_EMAIL_TEMPLATE = """
 Your request to land {landing_job_identifier} failed.
 
-See {lando_revision_url} for details.
+See {error_details_location} for details.
```

Comment:
Maybe we don't need this line at all, if there is no link? Instead of "see below". This would also simplify the logic in the rest of this file.